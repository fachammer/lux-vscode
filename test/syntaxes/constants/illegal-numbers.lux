1a
0b2
0xg
123.0e
123.4e+
123.4e-