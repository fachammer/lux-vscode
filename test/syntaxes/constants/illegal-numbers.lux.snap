>1a
#^^ source.lux invalid.illegal.numeric
>0b2
#^^^ source.lux invalid.illegal.numeric
>0xg
#^^^ source.lux invalid.illegal.numeric
>123.0e
#^^^^^^ source.lux invalid.illegal.numeric
>123.4e+
#^^^^^^^ source.lux invalid.illegal.numeric
>123.4e-
#^^^^^^^ source.lux invalid.illegal.numeric